crates/pathfinder/src/bin/pathfinder.rs

```diff
@@ -89,6 +91,12 @@ async fn main() -> anyhow::Result<()> {
             // This handle returns () so its not very useful.
             tracing::error!("RPC server process ended unexpected");
         }
+        result = update_handle => {
+            match result {
+                Ok(_) => tracing::error!("Release monitoring process ended unexpectedly"),
+                Err(err) => tracing::error!(error=%err, "Release monitoring process ended unexpectedly"),
+            }
+        }
```

Comment:
Should I leave this in? 

This means that if the update manitoring process fails, it will pull down the whole node. Should this be the case?

Reply:
I think this should happen quite rarely, but we'll see. Better to hide known errors later than to miss them without knowing about them in the first place.